app/src/shared/components/Button/newButton.pcss

```diff
@@ -0,0 +1,256 @@
+.root {
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  padding: 0;
+  background-color: transparent;
+  border: 1px solid;
+  border-radius: 4px;
+  font: var(--sans);
+  font-weight: var(--medium);
+  letter-spacing: 0;
+  text-align: center;
+  transition: 200ms ease-in-out;
+  transition-property: border-color, background-color, color, transform;
+  white-space: nowrap;
+  cursor: pointer;
+  user-select: none;
```

Comment:
We may wanna add `will-change: transform;` here to avoid them tiny 1px glitches on hover.